R/run_all_checks.R

```diff
@@ -45,28 +53,31 @@ run_all_checks <- function(metads = sdtmchecksmeta,
     
     if (!is.null(priority) & !all(priority %in% c("High", "Medium", "Low"))) {
         stop("priority argument should only take values 'High','Medium', or 'Low'")
-    }
-    if (!is.null(type) & !all(type %in% c("ALL", "ONC", "COVID", "PRO","OPHTH"))) {
+        }
+    if (!is.null(type) & !all(type %in% c("ALL", "ONC", "COVID", "PRO", "OPHTH"))) {
         stop("type argument should only take values 'ALL', 'ONC', 'COVID', 'PRO', 'OPHTH'")
-    }
+        }
     
     # subset meta dataset for specific Type and Priority
-    if(!is.null(priority)){
+    if (!is.null(priority)) {
         metads <-subset(metads,metads$priority %in% priority)
-    }
-    if(!is.null(type)){
-        metads <-subset(metads,metads$category %in% type)
-    }
+        }
+    
+    if (!is.null(type) ) {
+        metads <-subset(metads, metads$category %in% type)
+        }
     
     #Identify missing expected domains to cat out to the user later
-    expected.domains = sort(unique(strsplit(gsub(" ","",paste(unique(metads$domains),collapse=",")),",")[[1]]))
+    expected.domains = sort(unique(strsplit(gsub(" ", "", paste(unique(metads$domains),
+                                                                collapse=",")), ",")[[1]]))
     missing.domains = expected.domains[!(expected.domains %in% ls(envir = .GlobalEnv))]
     
     # Run checks
     
-    all_rec <- mcmapply(
+    all_rec <- parallel::mcmapply(
```

Comment:
@sarabodach I don't think you are supposed to source functions like this in a package, instead you import the function in Roxygen header, which I think is already being done